frontend/src/contexts/HostContext/HostContext.js

```diff
@@ -116,6 +116,15 @@ function HostProvider({ children }) {
     }
   }, [state.newPlayerStaging]);
 
+  useEffect(() => {
+    if (state.playerIDs.length === 1 && state.newPlayerStaging.length === 0) {
+      dispatch({
+        type: 'GAME_OVER',
+        payload: { gameWinner: state.playerIDs[0], playerIDs: state.playerIDs },
+      });
+    }
+  }, [state.playerIDs]);
+
```

Comment:
Instead of in a `useEffect` would it be possible to hook into the player disconnect event to check this?

Reply:
This might be good to do in a future PR, once these changes are merged https://github.com/I3uckwheat/cards-of-carousal/pull/312

Then I'll be able to handle this stuff inside the player disconnect event inside of the hostReducerMiddleware